src/SFA.DAS.EmployerIncentives.Commands/ServiceCollectionExtensions.cs

```diff
@@ -107,7 +108,8 @@ public static IServiceCollection AddCommandHandlerDecorators(this IServiceCollec
                 .Decorate(typeof(ICommandHandler<>), typeof(CommandHandlerWithDistributedLock<>))
                 .Decorate(typeof(ICommandHandler<>), typeof(CommandHandlerWithRetry<>))
                 .Decorate(typeof(ICommandHandler<>), typeof(CommandHandlerWithValidator<>))
-                .Decorate(typeof(ICommandHandler<>), typeof(CommandHandlerWithLogging<>));
+                .Decorate(typeof(ICommandHandler<>), typeof(CommandHandlerWithLogging<>))
+                .Decorate(typeof(ICommandHandler<>), typeof(CommandHandlerWithUnitOfWork<>));
```

Comment:
I think this should be the inner decorator.  So the order would be 

CommandHandlerWithLogging -> CommandHandlerWithValidator -> CommandHandlerWithRetry -> CommandHandlerWithDistributedLock -> CommandHandlerWithUnitOfWork -> handler

So needs to be moved to the first in the list.